include/ck/tensor_operation/gpu/device/impl/device_gemm_xdl_cshuffle_v3r1.hpp

```diff
@@ -36,6 +36,7 @@ template <typename ALayout,
           typename CDataType,
           typename GemmAccDataType,
           typename CShuffleDataType,
+          typename ReduceDataType,
```

Comment:
We can move `ReduceDataType` to the end and set default value as `typename ReduceDataType = CDataType`.

Reply:
done